docs/src/pages/api/00_usage.md

```diff
@@ -2,86 +2,197 @@
 title: "Usage"
 ---
 
-### Node
+Import the Octokit constructor.
 
-Install with `npm install @octokit/rest`.
+For direct usage in browsers, download `octokit-rest.min.js` from the [latest release](https://github.com/octokit/rest.js/releases/latest) and import it using a `<script src="octokit-rest.min.js"></script>` tag.
 
 ```js
-const octokit = require('@octokit/rest')()
+const Octokit = require('@octokit/rest')
```

Comment:
Is it also possible to use the `import Octokit from 'octokit/rest'`? style of import?

Reply:
I'm also interested on this.